src/Extensions/Diagnostics.HealthChecks/Internal/HttpHealthCheckParameters.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	internal class HttpHealthCheckParameters
+	{
+		public string EndpointName { get; }
+		public Uri RelativeUri { get; }
+		public HttpMethod Method { get; }
+		public HttpStatusCode ExpectedStatus { get; }
+		public string Scheme { get; }
+		public Func<HttpResponseMessage, HealthCheckResult, HealthCheckResult>? AdditionalCheck { get; }
```

Comment:
>AdditionalCheck [](start = 74, length = 15)

please add some comments about the properties, if possible including examples

Reply:
I've added a comment with the reference to the public method that has information about all of those fields